src/pages/guides/optimizing-developer-onboarding.tsx

```diff
@@ -0,0 +1,55 @@
+import { FunctionComponent } from 'react'
+
+import { Layout, GuideLayout } from '@components'
+
+export const Guide: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'Optimizing developer onboarding: Lessons learned and secrets to success from 31 companies and open source projects',
+            description:
+                'Learn about challenges open source projects and private companies face onboarding developers and best practices to onboard new developers more effectively.',
+        }}
+        className="navbar-white"
+    >
+        <GuideLayout
+            title="Optimizing developer onboarding"
+            subtitle="Lessons learned and secrets to success from 31 companies and open source projects"
+            // speakers={speakers}
+            form={{
+                formId: 'af3bc75e-c4f0-43e7-9829-c1d5d59e0a56',
+                onFormSubmitted: () =>
+                    window.open('/guides/optimizing-developer-onboarding/optimizing-developer-onboarding.pdf'),
```

Comment:
One last thing I noticed – it looks like they have this listed under `/guides/optimizing-developer-onboarding.pdf` for the link coming back from HubSpot's successful form submission message. Could you move the file to the root of `/guides`? If there are more static files necessary for this page, a directory would make sense in that case.

```suggestion
                    window.open('/guides/optimizing-developer-onboarding.pdf'),
```

Reply:
Sure, this makes sense! The pdf has a new home in ce577da.